dockerfiles/theia/src/entrypoint.sh

```diff
@@ -81,7 +81,8 @@ fi
 shopt -u nocasematch
 
 # run Che-Theia
-PROJ_ROOT=${CHE_PROJECTS_ROOT:-/projects}
+PROJ_ROOT=${PROJECTS_ROOT:-}
+[[ -z "$PROJ_ROOT" ]] && PROJ_ROOT=${CHE_PROJECTS_ROOT:-/projects}
```

Comment:
As far as I understand `PROJECTS_ROOT` and `CHE_PROJECTS_ROOT` can be set in devfile?

Reply:
DevWorkspace engine provides only `PROJECTS_ROOT` environment variable
Che-server, only `CHE_PROJECTS_ROOT`

if you use a devfile v1, then you can reference `CHE_PROJECTS_ROOT`, for a devfile v2, `CHE_PROJECTS_ROOT`

Reply:
Got it! Thanks!